### DISCUSSION TITLE
FlashAir product availability

### DISCUSSION THREAD

**Discussion by emoacht** [emoacht/SnowyImageCopy#62] [General]
I know Toshiba discontinued FlashAir in early 2019.
https://www.mouser.com/PCN/Toshiba_Memory_America_Inc_EOL_Notification_FlashAir_Jan_15_2019.pdf

So I don't have to wait for the W-05 generation anymore.

Now I have a powerful app like Snowy Plus, but I can't get FlashAir anymore.

So too sad.

_Originally posted by @dentphoto in https://github.com/emoacht/SnowyImageCopy/issues/58#issuecomment-735385847_

**Comment by emoacht** [comment]
Still there are updates
https://personal.kioxia.com/en-emea/software/flashair-configuration.html
https://personal.kioxia.com/en-emea/news.1.html

_Originally posted by @71m383nd3r in https://github.com/emoacht/SnowyImageCopy/issues/58#issuecomment-735398398_

**Comment by emoacht** [comment]
I can only say that the market conditions surrounding FlashAir have drastically changed since its launch in 2012.

**Comment by krist0ph3r** [comment]
Not sure what's the legal implications, but could anyone share any executable/compilable source for the windows app? It's ridiculous that they stop hosting the binary on their site!
